tasks/typeScript.js

```diff
@@ -1,12 +1,58 @@
-var gulp = require('gulp'),
-  changed = require('gulp-changed'),
+var fs = require('fs'),
```

Comment:
Being that this deals more with js than typescript or angular - is a different file and/or task name not more appropriate?
I see this as more of a versioning abstraction.